inspirehep/modules/workflows/tasks/merging.py

```diff
@@ -95,3 +97,33 @@ def merge_articles(obj, eng):
         obj.extra_data['callback_url'] = \
             get_resolve_merge_conflicts_callback_url()
     obj.save()
+
+
+def conflicts_ticket_context(user, obj):
+    server_name = current_app.config['SERVER_NAME']
+    workflow_id = obj.id
+    arxiv_ids = get_value(obj.data, 'arxiv_eprints.value') or []
+    for index, arxiv_id in enumerate(arxiv_ids):
+        if arxiv_id and is_arxiv_post_2007(arxiv_id):
+            arxiv_ids[index] = 'arXiv:{0}'.format(arxiv_id)
+    report_numbers = get_value(obj.data, 'report_numbers.value') or []
+    dois = [
+        "doi:{0}".format(doi)
+        for doi in get_value(obj.data, 'dois.value') or []
+    ]
+    recid = obj.extra_data.get('recid')
+    subject = ' '.join(filter(
+        lambda x: x is not None,
+        arxiv_ids + dois + report_numbers + ['(#{0})'.format(recid)]
+    ))
+
+    subject = "Merge conflict on workflow {} needs to be resolved. {}".format(workflow_id, subject)
```

Comment:
leftover

Reply:
Not really leftover. It's merging subject generated above with this text. If you prefer I can leave only thing generated above.